src/components/usa-button/content/index.js

```diff
@@ -0,0 +1,8 @@
+export { default as Data } from "./usa-button.json";
+export { default as AccentCoolData } from "./usa-button~accent-cool.json";
+export { default as AccentWarmData } from "./usa-button~accent-warm.json";
+export { default as BaseData } from "./usa-button~base.json";
+export { default as BigData } from "./usa-button~big.json";
+export { default as OutlineInverseData } from "./usa-button~outline-inverse.json";
```

Comment:
I believe you are missing an export for the Outline story (I see the json file for it but no reference here)